lib/src/rows/Rows.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import styled from "styled-components";
+import { RowProps, RowsProps } from "./types";
+
+const DxcRows = ({
+  alignX = "stretch",
+  alignY = "stretch",
+  as = "div",
+  divider = false,
+  gutter = "0rem",
+  reverse = false,
+  children,
+}: RowsProps): JSX.Element => {
+  return (
+    <RowsContainer as={as} alignX={alignX} gutter={gutter} reverse={reverse}>
+      {React.Children.map(children, (child, index) => {
+        return (
+          <>
+            {React.cloneElement(child, { alignY })}
+            {divider && index !== React.Children.count(children) - 1 && <Divider />}
+          </>
+        );
+      })}
+    </RowsContainer>
+  );
+};
+
+const DxcRow = ({ height = "auto", alignY, children }: RowProps): JSX.Element => {
+  return (
+    <StyledRow height={height} alignY={alignY}>
+      {children}
+    </StyledRow>
+  );
+};
+
+DxcRows.Row = DxcRow;
+
+const RowsContainer = styled.div<RowsProps>`
+  display: flex;
+  ${({ alignX, gutter, reverse }) => `
+    flex-direction: ${reverse ? "column-reverse" : "column"};
+    align-items: ${alignX};
+    gap: ${gutter};
+  `}
+  flex-grow: 1;
```

Comment:
Should we use `flex: 1` instead of `flex-grow: 1`? I'm not sure about the impact of `flex-basis` being set to auto with our current option...